homeworks/hw10/positive_func_args/hw10_solution.py

```diff
@@ -0,0 +1,13 @@
+def validate_arguments(func):
```

Comment:
Type hinting for decorators https://buutticonsulting.com/en/blog/2024/11/26/type-hinting-python-decorators-part-1/